nixos/lib/testing-python.nix

```
@@ -158,7 +160,7 @@ rec {
             mkdir -p $out/bin
 
             echo -n "$testScript" > $out/test-script
```

Hm, what about doing something like this instead:

```suggestion
            echo "# fmt: off" > $out/test-script
            echo -n "$testScript" >> $out/test-script
```

I do this by default on all my tests and if I'm not missing something, this causes black to not apply formatting but still does things such as syntax checks and other rather useful things.

---

What else does it do? I've re-read the README, but it only seems to talk about formatting.

---

Giving this a shot here https://github.com/NixOS/nixpkgs/pull/122199

---

@roberth: Gave this a shot as well here: https://github.com/NixOS/nixpkgs/pull/122201